docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md

```diff
@@ -144,9 +144,22 @@ cd darkflow
 
 To convert YOLOv1 or YOLOv2 model to TensorFlow, go to the root directory of the cloned DarkFlow repository and run the following command:<br>
 ```sh
-python3 ./flow --model <path_to_model>/<model_name>.cfg --load <path_to_model>/<model_name>.weights --savepb
+python3 flow --model <path_to_model>/<model_name>.cfg --load <path_to_model>/<model_name>.weights --labels <path_to_dataset_labels_file> --savepb
 ```
 
+For Yolo V1 `--labels` argument can be skipped:
+```sh
+python3 flow --model yolov1.cfg --load yolov1.weights --savepb
+```
+
+For Yolo V2 with VOC dataset additional changes in the original exporting script are required. 
+In the file`darkflow/utils/loader.py` change line 121 from `self.offset = 16` to `self.offset = 20` 
+And then run:
+```sh
+python3 flow --model yolov2-voc.cfg --load yolov2-voc.weights --labels voc-labels.txt --savepb
+```
+voc-labels can be found by the following link https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt
```

Comment:
Should not we mark this as hyperlink? Something like

`voc-labels can be found by the following [link](https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt)`

Reply:
The link is not 'official' so I wanted user to the full path. 
If i don't write [](https://) Markdown still shows it as a link but I can explicitly specify 
```[https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt](https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt)```


if we replace to see a short link, e.g. ```[short_link_name](https://raw.githubusercontent.com/szaza/android-yolo-v2/master/assets/tiny-yolo-voc-labels.txt)``` I suspect user may scare that we are trying to 'fish` him. Do you insist me to write a short link?